solution/ui/regulations/js/api.js

```diff
@@ -397,7 +397,12 @@ const v3GetSupplementalContent = async (apiURL, {locations, locationDetails=fals
 
 }
 
-// API Functions Insertion Point (do not change this text, it is being used by hygen cli)
+const getSubpartTOC = async (apiURL, title, part, subPart) => 
+{
+    const url = apiURL.replace('/v2/', '/v3/')
+
+    return httpApiGetLegacy(`${url}title/${title}/part/${part}/version/latest/subpart/${subPart}/toc`)
+}// API Functions Insertion Point (do not change this text, it is being used by hygen cli)
```

Comment:
Can we please get a new line added here?

Reply:
done